### PR TITLE
feat(donation modal): add English translation

### PR DESCRIPTION
**Pull request by yontank**
# Description
translates DonateModal to English when switching languages, also change Donate modal from LTR to RTL depending on the language specification

### CODE REVIEW

File: src/pages/DonateModal/DonateModal.tsx [hunk 1]
```diff
@@ -27,36 +26,33 @@ export const DonateModal: React.FC<DonateModalProps> = ({ isVisible, onClose })
    * @description This hook is used to determine if the image should collapse or not based on if the user is a mobile, tablet user or not.
    */
   const shouldImgCollapse = useMediaQuery('(max-width: 950px)')
+  const { t } = useTranslation()
 
   return (
     <Modal
+      dir={i18n.dir()}
```
Comment (NoamGaash) [inline src/pages/DonateModal/DonateModal.tsx:33]: is there a way to use global css rules to achive this behavior without inserting inline style to individual components?
If you don't know, I'd appreciate if you can open an issue to investigate it after we'll merge this one

**Review by NoamGaash — APPROVED** [review]
Thank you very much!

**Comment by NoamGaash** [comment]
run `npm run lint:fix` or `npm run lint -- --fix` to fix lint errors :)

File: src/pages/DonateModal/DonateModal.tsx (hunk 1 above)
Comment (yontank) [reply src/pages/DonateModal/DonateModal.tsx:33]: Not sure, Might be better to open an Issue about it but I can probably learn and do it alone

**Comment by NoamGaash** [comment]
small comment -
image

the colon (:) is aligned to the left instead to the right (should come directly after the field name)
The other changes (חשבון vs החשבון) are completely fine :)
Thank you very much!

**Review by yontank — COMMENTED** [review]
> small comment -
> image
> 
> the colon (:) is aligned to the left instead to the right (should come directly after the field name)
> The other changes (חשבון vs החשבון) are completely fine :)
> Thank you very much! 

yeah you're right, my bad!

**Review by yontank — COMMENTED** [review]
> Not sure, Might be better to open an Issue about it but I can probably learn and do it alone 

Yeah, I don't really know what to do here 🤦‍♂️, I'll open an Issue and merge it with the fixes and write some tests for the Modal.

Does everything look ok for a merge?

File: src/pages/DonateModal/DonateModal.tsx (hunk 1 above)
Comment (NoamGaash) [reply src/pages/DonateModal/DonateModal.tsx]: I think it can be nice of you if you can do both (open an issue and assign yourself) - up to you :)

**Review by NoamGaash — APPROVED** [review]
Perfect, thank you! :clap: :100: :bus:

**Pull request merged**